Python/Product/PythonTools/visualstudio_py_util.py

```diff
@@ -134,7 +134,9 @@ def exec_module(module, global_variables):
 def read_bytes(conn, count):
     b = to_bytes('')
     while len(b) < count:
-        b += conn.recv(count - len(b))
+        received_data = conn.recv(count - len(b))
+        if received_data:
+            b += received_data
```

Comment:
That was my assumption as well, and in that case we probably want:

```python
if received_data is None:
    break
```

Otherwise we could get stuck in an infinite loop here. If we're actually getting `b''` back then we'll still be stuck in the loop, but that was the prior behavior anyway and it hasn't been a problem.